### ISSUE TITLE
Development Issues: (manual) updates to 1.11.2 cause various errors

### ISSUE THREAD

**Issue by queengooborg** [TheCBProject/CodeChickenLib#46]
So, I've been working on updating a handful of mods from 1.10.2 to 1.11.2, CodeChickenLib included (although I noticed that it's already updated, after I started the process, durr...).  Admittedly, this is my first experience with Java (I mainly code in Python and C++), let alone Minecraft modding, so I'm probably missing something really simple along the way.  However, I've noticed that a lot of classes in CCL reference private class members from Minecraft itself, and as such I'm unable to compile the mods error-free.  I'm stuck on `codechicken.lib.render.state.GlStateManagerHelper`, where it seems that everything references private members and functions belonging to `GlStateManager`.

Bottom line, there has to be some important differences I'm missing between the compiler settings I have vs. the compiler settings used to create the official 1.11.2 release.  I want to know, what am I doing wrong?

Minecraft: 1.11.2
JRE: jre1.8.0_60 / jre1.8.0_121
Compiler Compliance Level: 1.8
Forge: 1.11.2-13.20.0.2260
IDE: Eclipse Java Neon (macOS/Windows)
Added buildscript repository: http://files.minecraftforge.net/maven (Gradle fails to obtain fernflower:2.0-SNAPSHOT without it)

**Comment by queengooborg** [comment]
I figured out what it was that caused all the issues: it turns out that Forge 13.20.0.2252 is the required version, and updating to 2260 messed everything up.  Oops!
